packages/@sanity/portable-text-editor/src/editor/plugins/__tests__/withPortableTextMarkModel.test.tsx

```diff
@@ -203,7 +203,7 @@ describe('plugin:withPortableTextMarksModel', () => {
             focus: {path: [{_key: 'a'}, 'children', {_key: 'a1'}], offset: 0},
             anchor: {path: [{_key: 'b'}, 'children', {_key: 'b1'}], offset: 1},
           })
-          PortableTextEditor.toggleMark(editor, 'bold')
+          PortableTextEditor.toggleMark(editor, 'strong')
```

Comment:
Or.. is this because now it will be removed because "bold" is not a valid mark? Question, how do you ensure that a user who is doing local development of the schema won't lose it's changes if another user is at the same place but with the old schema?

Reply:
Correct. The bold was never supported by the schema in the first place 😅

Reply:
The normalization done by the editor will only apply when the node is edited. That's why this normalization is only done through the editor, and not validated by validateValue.